### PR TITLE
add pytest to GitHub Actions

### LINKED ISSUE #137: Add pytest to github actions
### Expected Behavior

Add pytest to github actions

### Current Behavior

Only style testing is done as of the moment.

### PR DESCRIPTION
**Pull request by codeboy5**
# Pull Request

## Description
Closes #137 . Support has been added for running pytest on the PR.

## Affected Dependencies
List any dependencies that are required for this change.

## Type of Change
Please mark options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation (non-breaking change which adds documentation)
- [x] Improvement (non-breaking change that improves the performance or reliability of existing functionality)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [x] I did follow the [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I did follow the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md)
- [] I have added tests for my changes

## Additional Context
Please also include relevant motivation and context.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -42,11 +42,14 @@ jobs:
         bazel build src/python:bindings_test  --verbose_failures
         rm -f pydp.so
         cp -f ./bazel-bin/src/bindings/pydp.so ./pydp
+        python setup.py bdist_wheel
+        pip install dist/*.whl
     - name: Run tests
       run: |
         pip install pycodestyle
         pip install pytest
         python setup.py test
```
Comment (chinmayshah99) [inline .github/workflows/tests.yml:51]: Can we integrate this with setup.py test, rather than separate pytest?

Comment (codeboy5) [reply .github/workflows/tests.yml:51]: I'll look into it

**Comment by chinmayshah99** [comment]
Waiting for #158 to be fixed before we can merge this.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (chinmayshah99) [reply .github/workflows/tests.yml:51]: Any update on this @codeboy5

**Comment by chinmayshah99** [comment]
#193 Fixes this and a lot more.

**Pull request closed without merging**